ui/packages/app/src/components/Accordion/Accordion.svelte

```diff
@@ -14,14 +14,39 @@
 
 <div
 	id={elem_id}
-	class="p-3 border border-gray-200 dark:border-gray-700 rounded-lg flex flex-col gap-3 hover:border-gray-300 dark:hover:border-gray-600 transition"
-	class:hidden={!visible}
+	class="container border-gray-200 dark:border-gray-700 hover:border-gray-300 dark:hover:border-gray-600 "
+	style:display={visible ? "block" : "none"}
 >
-	<div on:click={toggle} class="w-full flex justify-between cursor-pointer">
+	<div on:click={toggle} class="label-wrap">
 		<span>{label}</span>
-		<span class:rotate-90={!_open} class="transition">▼</span>
+		<span style:transform={_open ? "rotate(0)" : "rotate(90deg)"} class="icon"
+			>▼</span
+		>
 	</div>
 	<Column visible={_open}>
 		<slot />
 	</Column>
 </div>
+
+<style>
+	.container {
+		display: flex;
+		flex-direction: column;
+		gap: var(--size-3);
+		padding: var(--size-3);
```

Comment:
Typeable value of slider in `blocks_update` is no longer centered

### This branch
![image](https://user-images.githubusercontent.com/41651716/209226753-317de2f3-34dd-4d8e-9fa5-ef006d16b2ee.png)


### 3.15.0
![image](https://user-images.githubusercontent.com/41651716/209226709-910a2f79-afb8-46c2-808b-010d4687a375.png)